### DISCUSSION TITLE
Delay between 2 tasks of a chain

### DISCUSSION THREAD

**Discussion by hmarty-ea** [django-q2/django-q2#145] [Q&A] [answered]
When creating chained subtasks using the async_chain() function, what is the delay between the end of one chained subtask and the next one? This is not clear to me.

**Comment by GDay** [comment, marked as answer]
Depends a bit on your settings, but if you haven't changed the `poll` and `guard_cycle` settings, then it should start very quickly, within a second (if all workers are not busy, of course).

**Reply by hmarty-ea** [reply]
Thanks! I was confused but in the end found that the reason was my tasks spending a lot more time than I expected!
